fastapi/openapi/utils.py

```diff
@@ -109,6 +109,10 @@ def get_openapi_operation_parameters(
         if field_info.deprecated:
             parameter["deprecated"] = field_info.deprecated
         parameters.append(parameter)
+        if isinstance(field_info, Query):
+            if not field_info.explode:
```

Comment:
What is the point of nested if statement here?

Reply:
I assume you're suggesting to merge these two conditions like this?
```python
if isinstance(field_info, Query) and not field_info.explode:
   parameter["explode"] = field_info.explode
   parameter["style"] = field_info.style.value
```

I can't see a reason why I chose nested version over this one. Maybe it was left after some refactoring. I can change that and update the branch as there are some conflicts with master.

Reply:
> I assume you're suggesting to merge these two conditions like this?
Yes